fip-0026.md

```diff
@@ -328,8 +397,9 @@ This sets the marketplace configurations.
 |owner_public_key|Yes|FIO Public Address|Valid public key of owner|
 |commissionfee|Yes|Number 0-25|Used to calculate commission fee from sale price|
 |listingfee|Yes|Number less than 10|Flat fee in FIO taken up front for listing a domain for sale|
-|duration|Yes|Number less than 365|Amount of time the listing will last, in days|
-|warn_days|Yes|Number less than `duration`-1|Number of days before expiration notification can be sent to the seller|
+|max_fee|Yes|Positive Int|Maximum amount of SUFs the user is willing to pay for fee. Should be preceded by [/get_fee](https://developers.fioprotocol.io/api/api-spec/reference/get-fee/get-fee) for correct value.|
+|tpid|Yes|FIO Address|FIO Address of the entity which generates this transaction. TPID rewards will be paid to this address. Set to empty if not known.|
+|actor|Yes|FIO account|Name of account to pay fee|
```

Comment:
What's the difference between owner and actor? Also same as above if only BPs can call let's remove fee and tpid.

`listingfee` - is this in addition or instead of commission? Should be explained above how commission is calculated. Also, convention is to use SUFs.